data/sql/updates/pending_db_world/rev_1573682333945401103.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1573682333945401103');
+
+DELETE FROM `gameobject` WHERE `id` IN (2061,2066);
```

Comment:
why delete all game objects flame ?

Reply:
Please read the explanations in my comments above.